qa/suites/upgrade-clients/client-upgrade-reef-tentacle/client-upgrade-reef-tentacle/tentacle-client-x/rbd/0-cluster/start.yaml

```diff
@@ -0,0 +1,21 @@
+meta:
+- desc: |
+   Install and run ceph on one node,
+   with a separate client 1.
+   Upgrade client 1 to reef
```

Comment:
```suggestion
   Upgrade client 1 to tentacle
```

Reply:
This extends to the existing `client-upgrade-reef-squid`.  It would be good to fix the corresponding comment there as well.